packages/web-console/src/scenes/Schema/Table/index.tsx

```diff
@@ -160,6 +161,24 @@ const Table = ({
     }
   }
 
+  const showMatViewDDL = async (name: string) => {
+    try {
+      const response = await quest.showMatViewDDL(name)
+      if (response && response.type === QuestDB.Type.DQL) {
+        return response
+      }
+    } catch (error: any) {
+      dispatch(
+        actions.query.addNotification({
+          content: (
+            <Text color="red">Cannot show DDL for materialized view '{name}'</Text>
```

Comment:
```suggestion
            <Text color="red">Cannot show query for materialized view '{name}'</Text>
```

If agreed, flagging here too.